public/app/plugins/datasource/prometheus/querybuilder/components/PromQueryEditorSelector.tsx

```diff
@@ -87,19 +78,26 @@ export const PromQueryEditorSelector = React.memo<PromQueryEditorProps>((props)
             />
           </>
         )}
+        <QueryHeaderSwitch label="Preview" value={query.queryPreview} onChange={onQueryPreviewChange} />
```

Comment:
Shouldn't we unify the name in toggle (Preview -> Query preview) with label (Query text -> Query preview)?

```suggestion
        <QueryHeaderSwitch label="Query preview" value={query.queryPreview} onChange={onQueryPreviewChange} />
```